src/ActionBuilder/ActionBuilderFactory.php

```diff
@@ -79,11 +79,6 @@ public function getActionBuilders(ClassMetadataInterface $classMetadata, string
             return $builder->supports($fieldPath, $sourceClass);
         });
 
-        // TODO Test
```

Comment:
The current implementation of priority doesn't work, because:
1. ... usort is wrong. Currently will be the highest builder executed at last (Interface says something different)
2. ... the changed data handle's the order of executed builder, not the priority tag. 

We refactored the priority handling. So we can control which action should be first or last. In example: The action for sorting attributes must be the last. The action for adding and removing attributes must be before the sorting action.